tests/amy/test_amy.py

```diff
@@ -374,3 +398,1223 @@ def _generate_day_from_index(i: int) -> str:
+            columns=columns,  # Same columns
+            index=range(1, 366),
+        )
+        mock_historic_profile.index.name = "Day of Year"
+
+        mock_compute_profile.side_effect = [mock_future_profile, mock_historic_profile]
+
+        # Execute - this should use direct element-wise subtraction
+        with patch("builtins.print") as mock_print:
+            result = get_climate_profile(warming_level=[2.0])
+
+        # Verify success message was printed
+        printed_output = [str(call) for call in mock_print.call_args_list]
+        success_found = any(
+            "Columns match - computing element-wise difference" in output
+            for output in printed_output
+        )
+        assert (
+            success_found
+        ), "Should have printed success message about matching columns"
+
+        # Should return DataFrame with same structure as input
+        assert isinstance(result, pd.DataFrame)
+        assert result.shape == (365, 24)
+        assert result.index.name == "Day of Year"
+
+    @pytest.mark.advanced
+    @patch("climakitae.explore.amy.retrieve_profile_data")
+    @patch("climakitae.explore.amy.compute_profile")
+    def test_get_climate_profile_future_multiindex_historic_simple_mismatch(
+        self, mock_compute_profile, mock_retrieve_profile_data
+    ):
+        """Test MultiIndex future with simple historic having hour mismatch."""
+        # Setup mocks
+        historic_data, future_data = self.create_mock_datasets()
+        mock_retrieve_profile_data.return_value = (historic_data, future_data)
+
+        # Future profile with MultiIndex (Hour, Warming_Level)
+        hours = range(1, 25)
+        warming_levels = ["WL_2.0"]
+        column_tuples = [(hour, wl) for hour in hours for wl in warming_levels]
+        multiindex_cols = pd.MultiIndex.from_tuples(
+            column_tuples, names=["Hour", "Warming_Level"]
+        )
+
+        mock_future_profile = pd.DataFrame(
+            np.random.rand(365, len(column_tuples)),
+            columns=multiindex_cols,
+            index=range(1, 366),
+        )
+        mock_future_profile.index.name = "Day of Year"
+
+        # Historic profile with different simple columns (e.g., string labels)
+        mock_historic_profile = pd.DataFrame(
+            np.random.rand(365, 24),
+            columns=[f"hour_{i}" for i in range(1, 25)],  # Different column names
+            index=range(1, 366),
+        )
+        mock_historic_profile.index.name = "Day of Year"
+
+        mock_compute_profile.side_effect = [mock_future_profile, mock_historic_profile]
+
+        # Execute - this should handle the hour mismatch with fallback logic
+        result = get_climate_profile(warming_level=[2.0])
+
+        # Should complete successfully despite hour mismatch
+        assert isinstance(result, pd.DataFrame)
+        assert result.shape[0] == 365
+
+
+# =========================== PROFILE FORMATTING TESTS ==============================
```

Comment:
These tests failing for me since `_format_profile_df_multi_wl` doesn't exist in AMY